pkgs/by-name/ki/kimai/package.nix

```diff
@@ -0,0 +1,61 @@
+{ php, fetchFromGitHub, lib, nixosTests }:
+
+php.buildComposerProject (finalAttrs: {
+  pname = "kimai";
+  version = "2.24.0";
+
+  src = fetchFromGitHub {
+    owner = "kimai";
+    repo = "kimai";
+    rev = finalAttrs.version;
+    hash = "sha256-C6i263sAfZwZELUIcZh/OmXZqgCKifjPYBafnH0wMC4=";
+  };
+
+  php = php.buildEnv {
+    extensions = ({ enabled, all }: enabled ++ (with all; [
+      gd
+      intl
+      mbstring
+      pdo
+      tokenizer
+      xml
+      xsl
+      zip
+    ]));
+
+    # Asset building and (later) cache building process requires a little bit
+    # more memory.
+    extraConfig = ''
+      memory_limit=384M
+    '';
+  };
+
+  vendorHash = "sha256-3y3FfSUuDyBGP1dsuzDORDqFNj3jYix5ArM+2FS4gn4=";
+
+  composerNoPlugins = false;
+  composerNoScripts = false;
+
+  postInstall = ''
+    # Make available the console utility, as Kimai doesn't list this in
+    # composer.json.
+    mkdir -p "$out"/share/php/kimai "$out"/bin
+    ln -s "$out"/share/php/kimai/bin/console "$out"/bin/console
+  '';
+
+  passthru.tests = {
+    kimai = nixosTests.kimai;
+  }
+
+  meta = {
+    description = "A web-based multi-user time-tracking application";
+    homepage = "https://www.kimai.org/";
+    license = lib.licenses.agpl3Plus;
+    longDescription = "
+      Kimai is a web-based multi-user time-tracking application. Works great for
+      everyone: freelancers, companies, organizations - everyone can track their
+      times, generate reports, create invoices and do so much more.
+    ";
+    maintainers = lib.teams.php.members;
```

Comment:
I don't *think* maintenance of this should just be assigned to the PHP team based on this software being written in PHP, but perhaps I'm unaware of outside information or PHP specific conventions here.

Usually you would first init yourself as a maintainer in [`/maintainers/maintainer-list.nix`](https://github.com/NixOS/nixpkgs/blob/a5337b56fe644823e36acb7d59859c50056a377d/maintainers/maintainer-list.nix), then init this package with yourself as the maintainer.

Reply:
I was copy-pasting this area of snippet and didn't think about it much. Added my name to the maintainer list and set maintainers of the package and the test to myself.